portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,14 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Fetches random quote from the server
```

Comment:
nit: Add period after server. Here are below. Comments should be complete senetences.